mesonbuild/dependencies/python.py

```diff
@@ -357,14 +357,19 @@ def __init__(self, name: str, environment: 'Environment',
                 self.find_libpy_windows(environment, limited_api=False)
             else:
                 self.find_libpy(environment)
+        elif self.platform.startswith('ios-'):
+            # iOS doesn't use link_libpython - it links with the *framework*.
+            self.link_args = ["-framework", "Python", "-F", self.variables.get('prefix')]
+            self.is_found = True
```

Comment:
- consistent (single) quotes?
- This only matches if link_libpython is false. That assumes that sysconfig defines `LIBPYTHON` as an empty string, which is e.g. true on Linux with python 3.13 and I suppose is true for iOS builds as well. Hopefully this is a guarantee, but, maybe it makes sense to do the iOS check first anyway, given that self.platform is more fine-grained and stable than "the Makefile defined LIBPYTHON for substitution in configured files such as python.pc and python-config.sh"